openapi/components/schemas/OrganizationExports/OrganizationExport.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  id:
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  userId:
+    description: The ID of the User who requested the organization data export.
+    readOnly: true
+    type: string
+  status:
+    description: Status of organization data export request.
+    readOnly: true
+    type: string
+    enum:
+      - pending
+      - processing
+      - completed
+      - queued
+      - failed
+  items:
+    description: Organization data export resources array.
+    readOnly: true
+    type: array
+    items:
+      $ref: ../OrganizationExports/OrganizationExportResource.yaml
+  createdTime:
+    description: The time the organization data export was requested.
+    readOnly: true
+    allOf:
+      - $ref: ../ServerTimestamp.yaml
+  updatedTime:
+    description: Organization data export updated time.
+    readOnly: true
+    allOf:
+      - $ref: ../ServerTimestamp.yaml
+  retentionTime:
+    description: Retention date when the files will be removed.
```

nitpick: suggestion for clarity
```suggestion
    description: Retention end date. After this date, files will be removed.
```